docs/ref/contrib/staticfiles.txt

```diff
@@ -313,13 +313,29 @@ For example, the ``'css/styles.css'`` file with this content:
 
     @import url("../admin/css/base.27e20196a850.css");
 
+You can store manifest file with code by subclassing
+``ManifestStaticFilesStorage`` and setting the ``manifest_storage`` attribute::
```

Comment:
Curious: why are we discussing setting an attribute here instead of using a keyword argument? Am I missing something?

Reply:
Changed `attribute` to `argument`.